src/renderer/auth/components/register/Register/Register.js

```diff
@@ -25,48 +18,44 @@ export default class Register extends React.PureComponent {
   };
 
   state = {
-    tab: TAB_CREATE
+    step: 2
```

Comment:
Should we replace the step values with variables such as `STEP_CREATE` in case we add intermittent steps in the future?